### ISSUE TITLE
Add description of new `use_async` to the README

### ISSUE THREAD

**Issue by AnnMarieW** [plotly/dash-component-boilerplate#130]
It's great that there's now an option to generate a project that does async loading of  components :confetti_ball: 

It would be helpful to add a short description to the `README.md` in section 3 " Answer the questions about the project." so people would know what this is and if they should select 'True' or 'False'.

**Comment by nlyle01** [comment]
+1, came here to say the same thing

**Comment by danielleos** [comment]
Agree, this would be really helpful! I'm coming from a React background with little knowledge of python and plotly so would love to know what this option means.
